ReadMe.md

```diff
@@ -55,6 +55,18 @@ To run the app in production build of the app, use either of the following.
 ```bash
  npm start
 ```
+## Running the Migrations
+
+```bash
+ sequelize db:migrate
```

Comment:
Can you add this into script? this is to make it easy yo run: 

Example: `npm run migrate`